gateway/ln-gateway/src/bin/ln_gateway.rs

```diff
@@ -29,8 +29,8 @@ use tokio::sync::mpsc;
 use tracing::error;
 
 /// Fedimint gateway packaged as a CLN plugin
+// Use CLN_PLUGIN_LOG=1 to enable debug logging from within cln-plugin
```

Comment:
I think it take same values as `RUST_LOG` (just under different name). https://github.com/ElementsProject/lightning/blob/acfb63e4bf19751167fbc69448f236ae413e690f/plugins/src/logging.rs#L56

Reply:
Ah cool. Give it a log level: https://docs.rs/log/latest/log/enum.Level.html